packages/@webex/plugin-meetings/src/reachability/clusterReachability.ts

```diff
@@ -59,16 +59,25 @@ export class ClusterReachability extends EventsScope {
   public subnets: Map<
     string,
     {
-      serverIps?: string;
-      port?: number;
-      protocol?: string;
-      reachable?: boolean;
-      'answered-tx'?: number;
-      'lost-tx'?: number;
+      serverIp: string;
+      port: number;
+      protocol: string;
+      reachable: 'true' | 'false';
+      'answered-tx': number;
+      'lost-tx': number;
       latencies: number[];
     }
   > = new Map();
 
+  // Convert `serverIp` to `serverIp` for TransportResultForBackend
+  public getBackendDetails(): SubnetDetails[] {
```

Comment:
what is the purpose of this method? it does nothing right now. the serverIp is changed to serverIps in processProtocol(), not here, so what's the point of this one here?